galaxy_importer/collection.py

```diff
@@ -137,11 +138,13 @@ def __init__(self, path, filename, cfg=None, logger=None):
 
         self.content_objs = None
         self.metadata = None
+        self.file_manifest_file = None
         self.docs_blob = None
         self.contents = None
 
     def load(self):
         self._load_collection_manifest()
+        self._load_collection_file_manifest()
```

Comment:
I was confused with `file_manifest_file` thinking it was `MANIFEST.json` not `FILES.json`

Reply:
Agreed. manifest, manifest_file, file_manifest, file_manifest_file kind of blur together.

`files_manifest_file` slightly better.
`files_file` seems slightly worse.

Trying to avoid  `files_json` or `files_json_file` since it's not necessarily a  json file (possibly in the future), but maybe the most obvious 'family feud rules' name.

Reply:
i would rather refactor `files_json` later if the format did change. having a method (and var names) called `_load_collection_file_manifest()` that is related to `FILES.json` _not_ the manifest file `MANIFEST.json` is confusing now to me and will be likely be more confusing in the future and make it hard to maintain.

I believe `manifest` var/methods should only reference `MANIFEST.json`

Reply:
never thought about 'family feud rules' haha, but yeah i think it applies here

Reply:
This is kind of further confused by MANIFEST.json not actually being a 'manifest' by any normal definition.

and of course a manifest file that is a manifest of manifest files...  brane hurt.

I think I have the logic and implementation mostly sorted now, so i'll revisit the names now.